File: app/src/main/java/com/example/sasham/testproject/network/MessageWraper.java
package com.example.sasham.testproject.network;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class MessageWraper {
    @SerializedName("objects")
    @Expose
    private List<MessageAnswer> messageAnswers;

    public List<MessageAnswer> getMessageAnswers() {
        return messageAnswers;
    }

    public void setMessageAnswers(List<MessageAnswer> messageAnswers) {
        this.messageAnswers = messageAnswers;
    }
}

File: app/src/main/java/com/example/sasham/testproject/network/MessageWraper.kt
package com.example.sasham.testproject.network

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class MessageWraper {
    @SerializedName("userWrappers")
    @Expose
    var messageAnswers: List<MessageAnswer>? = null
}
